.storybook/config.js

```diff
@@ -15,6 +15,17 @@ twigContext.keys().forEach((key) => twig({
 }));
 
 addParameters({
+  options: {
+    storySort: (a, b) => {
```

Comment:
By default, the order of the Storybook menu is alphabetical, which interposes "Grids" between "Atoms" and "Molecules". This ordering function moves "Grids" beneath all other entries, making the pattern hierarchy easier to read.
<img width="150" alt="Screenshot 2019-11-20 13 38 29" src="https://user-images.githubusercontent.com/2893480/69243478-121dca00-0b9b-11ea-914c-e5e871f25e3a.png">